arviz/tests/base_tests/test_plots_matplotlib.py

```diff
@@ -1159,3 +1160,20 @@ def test_plot_mcse_no_divergences(models):
     idata.sample_stats = idata.sample_stats.rename({"diverging": "diverging_missing"})
     with pytest.raises(ValueError, match="not contain diverging"):
         plot_mcse(idata, rug=True)
+
+@pytest.mark.parametrize(
+    "kwargs",
+    [
+        {},
+        {"var_names": ["theta"]},
+        {"var_names": ["mu"], "coords": {"theta_dim_0": [0, 1]}},
```

Comment:
var_names should be theta, otherwise it will either be ignored or raise an error

Reply:
I don't understand as all these vars are present in fake data generated in `helpers.py` and the test is working fine.

Reply:
My point is that `coords` is not doing anything, so it is not actually testing it works. You can see what happens is a similar situation with the eight schools model:

```
idata = az.load_arviz_data("centered_eight")
az.plot_pp(idata, var_names="mu", coords={"school": [0,1]})
# and compare with 
az.plot_pp(idata, var_names="mu", coords={"school": [3,7]})
```

Reply:
Okay I understand.

Reply:
I think we can add other tests after adding more groups or can you suggest some tests to add?